library/patient.inc.php

```diff
@@ -1822,3 +1822,17 @@ function updateDupScore($pid)
     );
     return $dupscore;
 }
+
+function get_unallocated_payment_id($pid)
```

Comment:
where is the `get_unallocated_patient_balance()` function which is used in above script?
(this is different where is `get_unallocated_payment_id()`)

Reply:
Ok, I found the `get_unallocated_patient_balance()` function which is already in the codebase.
So no my question is just what this `get_unallocated_payment_id()` function is needed for?

Reply:
just brought in e1b2a591163f30daba16248266e31f013083ee0f

Reply:
also just noted where get_unallocated_payment_id() is used above :)